modules/client-configuration/pages/clients-sleses.adoc

```diff
@@ -29,6 +29,13 @@ ifeval::[{uyuni-content} == true]
 endif::[]
 
 
+[IMPORTANT]
+====
+Traditional clients are not available on {rhel}{nbsp}8.
+{rhel}{nbsp}8 clients are only supported as Salt clients.
+====
+
+
```

Comment:
I don't see where it is explained how to import Appstream from the DVD at this document.

Reply:
It's not. We explicitly say that the reader needs to organise access to base media. I think we can add this in when we re-organise these pages later on, though.

Reply:
@Loquacity this is already explained step by step for 6 and 7 on this document.

So we need to explain it for 8.

Check the document below, my other comments, or with Joseph.

Reply:
I see it on L100?

Reply:
![image](https://user-images.githubusercontent.com/3914967/71083680-4e1a6e00-21df-11ea-869f-3b82f411e654.png)
(p45 in the PDF I attached)

Reply:
No, this is NOT what I was talking about. We are talking about **importing the DVD**.

This about adding the channels from SCC for the client tools and RES updates.

Reply:
But anyway @jcayouette fixed it and added the instructions for DVD import for RHEL/CentOS 8.